<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QWidget" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QLabel" name="registration_username_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>95</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="registration_password_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>165</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="re_password_line_edit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="back_to_login_pushbutton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回登陆界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="re_password_confirm_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>235</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>..黑体UI-日本语</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码确认</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="registration_pushbutton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="re_username_line_edit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="re_password_confirm_line_edit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
